### DISCUSSION TITLE
Static IP Address

### DISCUSSION THREAD

**Discussion by cweigl** [softwarecrash/EPEver2MQTT#19] [Q&A]
Hello,
i could not find at the Wifi settings to set up the ESP to a static ip address. I only found SSID, Password and mqtt settings. Is it possible ? If yes, please let me know how to do it.

Thank you so much for help

**Comment by softwarecrash** [comment]
Hello,
actual is the static IP function not implemented.
when you realy need it, make a feautere request

**Reply by cweigl** [reply]
Thank you for very fast reply!
